tableauserverclient/models/data_alert_item.py

```diff
@@ -0,0 +1,197 @@
+import xml.etree.ElementTree as ET
+
+from .property_decorators import property_not_empty, property_is_enum
+from .user_item import UserItem
+from .view_item import ViewItem
+
+
+class DataAlertItem(object):
+    class Frequency:
```

Comment:
The enums are intended to allow users to pass in `TSC.DataAlertItem.Frequency.Once`, rather than a string value. With the way the `@property_is_enum` annotation is implemented, the string values listed here have to match the enum names exactly for it to work (e.g. `Once = 'Once'`). But server only accepts lowercase values for this particular field, so we need to convert it to lowercase in request_factory.

Could you change these string values to start with capitalized letters and also convert frequency into lowercase in request_factory.py?

Reply:
Capital cased the string values.